perfeed/data_stores/utils.py

```diff
@@ -0,0 +1,34 @@
+from pydantic import BaseModel, ValidationError
+from perfeed.models.pr_summary import PRSummary, PRSummaryMetadata
+import pandas as pd
+import json
+
+
+def validate_and_convert(data_type: str, data: BaseModel, metadata: BaseModel, is_sqldb: bool=False) -> pd.DataFrame:
```

Comment:
I'd suggest moving this to `BaseStorage` class and change it to be like
`def validate_and_convert(data_type, data, metadata: BaseModel, is_sqldb: bool):` then you can just skip the following `if-condition` which will grow as we have more types:
```
    if data_type == 'pr_summary':
        data_model = PRSummary
        metadat_model = PRSummaryMetadata
    else:
        raise RuntimeError('data_type not supported.')
```

Reply:
that makes sense and u can save `is_sqldb` arg as well. But then ur will have 80% replicated code sits in storage_feather.py and storage_sqldb.py? made the commit, take a look